### PR TITLE
Added a github action 

### LINKED ISSUE #1116: Request to add a github action to the repo
Feature request :
This is an awesome and an engaging repository , plus due to hacktoberfest'21 many open source enthusiasts ( preferably students who are first time contributors ) open issues and PRs which eventually help repo to grow
I think there should be a github action to welcome contributors on their each issue and PR so as to make repository more interactive

If you love the idea , please assign this to me

### PR DESCRIPTION
**Pull request by Jassi10000**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/lukeautry/tsoa/tree/master/docs/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/lukeautry/tsoa/pulls) for the same update/change?
- [ ] Have you written unit tests?
- [ ] Have you written unit tests that cover the negative cases (i.e.: if bad data is submitted, does the library respond properly)?
- [x] This PR is associated with an existing issue?

**Closing issues**

Put `closes #XXXX` (where XXXX is the issue number) in your comment to auto-close the issue that your PR fixes.
closes #1116 
### If this is a new feature submission:

- [x] Has the issue had a maintainer respond to the issue and clarify that the feature is something that aligns with the [goals](https://github.com/lukeautry/tsoa#goal) and [philosophy](https://github.com/lukeautry/tsoa#philosophy) of the project?

**Potential Problems With The Approach**

**Test plan**

### CODE REVIEW

**Comment by Jassi10000** [comment]
@WoH Please review this

File: .github/workflows/welcome-first-time-contributors.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          Hello there ${{ github.actor }} 👋
+
+          Welcome to tsoa !!💖🥳
+
+          Thank you and congratulations 🎉 for opening your very first issue in this project. tsoa fosters an open and welcoming environment for all our contributors.🌸
+
+          Incase you want to claim this issue, please comment down below! We will try to get back to you as soon as we can.👀
+
+          👩‍💻 If you have any interesting ideas, just open an issue. We would love to hear you and engage in discussions.
+
```
Comment (WoH) [inline .github/workflows/welcome-first-time-contributors.yml:28]: I think a bot message for every first time someone opens an issue may be too noisy. Let's focus on a good PR message instead.

File: .github/workflows/welcome-first-time-contributors.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        pr-message: |
+          Hello there ${{ github.actor }} 👋
+
+          Thank you and congrats 🎉 for opening your first PR on this project.✨
+
+          tsoa fosters an open and welcoming environment for all our contributors.🌸
```
Comment (WoH) [inline .github/workflows/welcome-first-time-contributors.yml:34]: Don't think we should (have to) mention that explicitly.

**Comment by Jassi10000** [comment]
@WoH No issues
Just tell what changes you would love to see , I will make them right away !!!!

File: .github/workflows/welcome-first-time-contributors.yml (hunk 1 above)
Comment (WoH) [inline .github/workflows/welcome-first-time-contributors.yml:28]:
```suggestion

```

File: .github/workflows/welcome-first-time-contributors.yml (hunk 2 above)
Comment (WoH) [inline .github/workflows/welcome-first-time-contributors.yml:34]:
```suggestion

```

**Comment by WoH** [comment]
Added the comments as suggestions.

File: .github/workflows/welcome-first-time-contributors.yml (hunk 1 above)
Comment (Jassi10000) [reply .github/workflows/welcome-first-time-contributors.yml:28]: > 

This is removal of whole message 
What I should I add then  ?

Comment (WoH) [reply .github/workflows/welcome-first-time-contributors.yml:28]: Nothing in my opinion.
However, I'd be fine with a very short and sweet message like:

> Hello there ${{ github.actor }} 👋
          Thank you for opening your very first issue in this project. We will try to get back to you as soon as we can.👀

**Comment by Jassi10000** [comment]
@WoH Can you review the latest made changes

**Comment by WoH** [comment]
LGTM, thank you and congrats on your first PR here!

**Comment by Jassi10000** [comment]
> LGTM, thank you and congrats on your first PR here!

Thank you for the opportunity

**Pull request merged**